scripts/add_field_for_stuck_registrations/README.md

```diff
@@ -0,0 +1,11 @@
+Add Field For Contacts Stuck In The Registration Consent State
+----------------------
+The add_field_to_stuck_registration_contacts.py script takes in a CSV in stdin,
+and outputs a CSV on stdout. The input CSV is contacts in the stuck state
+since the implementation on asking for consent in a clinic prebirth & postbirth
+registration.
+The output CSV contains the same data as the input CSV, but with and additional
+field that notes that an sms should be sent to that contact to complete their
+registration.
```

Comment:
This is not true. What it does is call the RapidPro API to update the `stuck_in_consent_state` field on their contact

Reply:
I forgot to make changes to the readme, thanks for pointing this out